tests/ui/fmt/help-message-issue-129966-reorder-remove-format-specifier.rs

```diff
@@ -0,0 +1,14 @@
+// Test checks for the help messages in the error output/
+//@ check-run-results
+
+struct Foo(u8, u8);
```

Comment:
Problem [REV-RUSTFIX 2/4]: if we use `run-rustfix`, some lints fire like `dead_code` and checks for missing debug impl on `Foo`.

```rs
#![allow(dead_code)]

#[derive(Debug)]
struct Foo(u8, u8);
```